### ISSUE TITLE
Make neos/swiftmailer optional?

### ISSUE THREAD

**Issue by rolandschuetz** [Unikka/neos-linkchecker#18]
I think SwiftMailer could be an optional dependency, only needed when you use the main feature. What do you think?

**Comment by markusguenther** [comment]
Could make sense, but then I need to create another package that takes care of that.
Do you have issues with the mailer that is included or is it more to save dependencies?

**Comment by rolandschuetz** [comment]
No, not abort, I would just check [here](https://github.com/Unikka/neos-linkchecker/blob/master/Classes/Service/EmailService.php#L89) is the class exists, and it not throws an exception that the user needs to consider the optional dependency.
